experiments/snippets/skuidCustom/skuidCustom.js

```diff
@@ -0,0 +1,1344 @@
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-tabs */
+/* eslint-disable indent */
+/* eslint-disable semi */
+/* eslint-disable max-len */
+/* eslint-disable no-continue */
+/* eslint-disable no-shadow */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-extend-native */
+/* eslint-disable no-undef */
+const $ = skuid.$;
+
+// Custom SKUID Functions
+skuid.custom = {};
+
+// Date.formatDate(date)
+// formats a javascript date as YYYY-MM-DD for use with Salesforce
+Date.formatDate = function (date) {

```

Comment:
Skuid provides a utility for doing this out of the box: `skuid.time.getSFDate(jsDate)`
`https://docs.skuid.com/latest/v1/en/skuid/api/skuid_time.html#skuid.time.getSFDate

Reply:
Removed from skuidCustom